### PR TITLE
Improve key bindings

### PR DESCRIPTION
**Pull request by bjrnt**
Cool project!
Thought I'd try adding a couple of keybindings. :)
Also noticed that `esc` doesn't seem to do anything, so I updated the selector for that as well.

This PR:
- Adds <kbd>Ctrl/Cmd</kbd> + <kbd>1 ... 5</kbd> to navigate between tabs.
- Updates the selector for the `esc` binding. It now uses `aria-label="Close"`, which should work better than a CSS selector.

Would be grateful for any feedback!

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -62,24 +62,24 @@ function registerShortcuts(username) {
 		return false;
 	});
 
-	Mousetrap.bind('g h', () => {
+	Mousetrap.bind(['g h', 'command+1'], () => {
```
Comment (sindresorhus) [inline browser.js]: Should use `mod` instead of `command`. See the mousetrap docs.

File: browser.js [hunk 2]
```diff
@@ -95,8 +95,9 @@ function registerShortcuts(username) {
 		$(`a[href$="/${username}/lists"]`).click();
 	});
 
+	// Closes images, DM windows, etc.
 	Mousetrap.bindGlobal('esc', () => {
-		const btn = $('button._158OzO7l');
+		const btn = $('button[aria-label="Close"]');
```
Comment (sindresorhus) [inline browser.js]: This won't work for other localizations.

File: readme.md [hunk 3]
```diff
@@ -29,11 +29,11 @@ When closing the window, the app will continue running in the background, in the
...
-- Go to Home: <kbd>g</kbd> <kbd>h</kbd>
-- Go to Notifications: <kbd>g</kbd> <kbd>n</kbd>
-- Go to Messages: <kbd>g</kbd> <kbd>m</kbd>
-- Go to Search: <kbd>/</kbd>
-- Go to Profile: <kbd>g</kbd> <kbd>p</kbd>
+- Go to Home: <kbd>g</kbd> <kbd>h</kbd> or <kbd>Ctrl</kbd> <kbd>1</kbd>
+- Go to Notifications: <kbd>g</kbd> <kbd>n</kbd> or <kbd>Ctrl</kbd> <kbd>2</kbd>
+- Go to Messages: <kbd>g</kbd> <kbd>m</kbd> or <kbd>Ctrl</kbd> <kbd>3</kbd>
+- Go to Search: <kbd>/</kbd> or <kbd>Ctrl</kbd> <kbd>4</kbd>
+- Go to Profile: <kbd>g</kbd> <kbd>p</kbd> or <kbd>Ctrl</kbd> <kbd>5</kbd>
```
Comment (sindresorhus) [inline readme.md]: Need to document `Cmd` too.

**Comment by bjrnt** [comment]
@sindresorhus All done! :) Switched out the `aria-label` selector for a CSS one.

**Pull request merged**